### PR TITLE
Fix entity stay highlighted after hand leaves when far grabbing another

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
#### Test Plan

- Create two entities and make them grabbable.
- Move them a distance apart.
- Far grab one.
- Move your grabbing hand into the other entity. It highlights (as if you're able to near grab it).
- Move your grabbing hand out of that other entity.
- The highlight should be removed.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11896/HighFidelity-Beta-PR13767-c9c23b8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11896/HighFidelity-Beta-PR13767-c9c23b8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11896/HighFidelity-Beta-Interface-PR13767-c9c23b8.dmg

**Comment by qlimnavarro** [comment]
Currently Testing

File: scripts/system/controllers/controllerModules/nearParentGrabEntity.js [hunk 1]
```diff
@@ -322,6 +328,7 @@ Script.include("/~/system/libraries/controllers.js");
                 if (!props) {
                     // entity was deleted
                     unhighlightTargetEntity(this.targetEntityID);
+                    this.highlightedEntity = NULL;
```
Comment (thoys) [inline scripts/system/controllers/controllerModules/nearParentGrabEntity.js]: `NULL` does not exist in javascript please use lowercase `null`

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11922/HighFidelity-Beta-PR13767-a7170a5.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11922/HighFidelity-Beta-PR13767-a7170a5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11922/HighFidelity-Beta-Interface-PR13767-a7170a5.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Primitive highlights successfully.

**Pull request merged**